pkg/os/util_windows.go

```diff
@@ -0,0 +1,27 @@
+package os
+
+import (
+	"bytes"
+	"io/ioutil"
+
+	"golang.org/x/text/encoding/unicode"
+	"golang.org/x/text/transform"
+)
+
+func ReadFileUTF16(filename string) ([]byte, error) {
+	// Read the file into a []byte:
+	raw, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+
+	// Make an tranformer that converts MS-Win default to UTF8:
+	win16le := unicode.UTF16(unicode.LittleEndian, unicode.IgnoreBOM)
```

Comment:
Ignore byte order mark? Why ???

Reply:
This function is not documented so unsure why this is needed

Reply:
This is needed since the file created by secedit.exe /ecport is not utf-8 and when read directly without the conversion, the text are malformed.

Reply:
It's documented at https://godoc.org/golang.org/x/text/encoding/unicode#UTF16
`UseBOM` would probably be slightly more correct, unless we are sure Windows UTF16 files *never* start with a BOM.